src/pubsub/peer-streams.js

```diff
@@ -107,21 +109,21 @@ class PeerStreams extends EventEmitter {
       throw new Error('No writable connection to ' + id)
     }
 
+    // @ts-ignore - this.outboundStream could be null
```

Comment:
Relevant comment here
https://github.com/libp2p/js-libp2p-interfaces/pull/74#discussion_r534383557

Reply:
I changed the if clause to include the `outBound` stream and avoid the `ts-ignore`.
I do not think we should change all the internal codebase to fulfil every single type scenario, otherwise we would just do typescript. For the user based APIs I completely agree that we should make this effort, but `peer-streams` is an internal class for pubsub routers